meinberlin/apps/budgeting/rules.py

```diff
@@ -45,7 +46,10 @@
 
 rules.add_perm(
     "meinberlin_budgeting.moderate_proposal",
-    module_predicates.is_context_moderator | module_predicates.is_context_initiator,
+    is_superuser
+    | module_predicates.is_context_moderator
+    | module_predicates.is_context_initiator
+    | module_predicates.is_context_group_member,
```

Comment:
Couldnt we use ```is_allowed_moderate_project``` here? https://github.com/liqd/adhocracy4/blob/ca0331aa7612304691fe637903fec4f6936a1f9f/adhocracy4/modules/predicates.py#L87

Reply:
Yes! It does exactly do that. Will do.